tex/chapters/05-Implementation.tex

```diff
@@ -272,7 +299,11 @@ \subsection{Testing}
 
 Axelrod-Python uses several external programmes to help with testing.
 Travis \cite{Travis} provides continuous integration testing, so all tests are run automatically whenever any changes to code are made.
+% TODO Add a screenshot of all the tests passing on the PR?
```

Comment:
@drvinceknight The PR no longer shows this information (i assume because it's closed?) Should I take a screenshot of the main repo tests passing, or is there something from Travis/Appveyor that you had in mind?